README.md

```diff
@@ -59,3 +59,44 @@ The project currently has a `GET hello-world` Api Gateway<>Lambda integration se
    ```
    yarn test:integ
    ```
+
+## Webhook Quoting Schema
+
+Quoters will need to abide by the following schemas in order to successfully quote UniswapX orders.
+
+### Request
+
+This data will be included in the body of the request and will be sent to the given quote endpoint.
+
+```
+{
+   tokenInChainId: number,
+   tokenOutChainId: number,
+   requestId: string,
+   tokenIn: string,
+   tokenOut: string,
+   amount: string,
+   swapper: string,
+   type: string (ex. EXACT_INPUT or EXACT_OUTPUT),
```

Comment:
nit I'd remove the "ex" here to make it more explicit that it can only be those two string values